keps/prod-readiness/sig-storage/177.yaml

```diff
@@ -0,0 +1,3 @@
+kep-number: 177
+stable:
+  approver: "@deads2k"
```

Comment:
There are two KEPs that had PRR approvers set but weren't really prr-approved: 365 and 1867 - for those I didn't create corresponding files.

The others were created to keep the history.